src/game_options.cpp

```diff
@@ -170,11 +172,11 @@ void test_game_options()
     game_options go;
     assert(go.get_environment_type() == environment_type::empty);
   }
-//#define FIX_ISSUE_681
+#define FIX_ISSUE_681
 #ifdef FIX_ISSUE_681
   // (681) The game_options set a time limit
   {
-    const int max_time = 500; // change if too long/short
+    const int max_time = 6000; // change if too long/short
```

Comment:
That looks appropriate :)

Reply:
Might still be too short for one round, but that's not a thing we should think about for now